src/arrays/ArrayUtil.java

```diff
@@ -36,8 +66,17 @@ public static void create2dArray(int x, int y) {
      * @return true if the array contains the given value, otherwise return false.
      */
     public static boolean contains(int[] array, int value) {
-
-        return false;
+        boolean x = true;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value) {
+                x = true;
+                break;
+            } else {
+                x = false;
+            }
+        }
+        System.out.print(x);
+        return x;
     }
 
```

Comment:
It's ok too, for compact you can write this don't using break and else.
For example`
  boolean x = false;
        for (int i = 0; i < array.length; i++) {
            if (array[i] == value) {
                x = true;
                return x;
            }
        }
        return x;